File: c/udemy/7-matrizes/negativos_matriz.c
// Ler dois números M e N (máximo = 10), e depois ler uma matriz MxN de números inteiros, conforme exemplo. Em seguida, mostrar na tela somente os números negativos da matriz.

#include <stdio.h>

int main(){

    int m, n;
    printf("Linhas e colunas da matriz (max 10): ");
    scanf("%d %d",&m, &n);

    int mat[m][n];

    for (int i=0; i<m; i++){
        for (int j=0; j<n; j++){
            printf("Elemento [%d][%d]: ", i, j);
            scanf("%d",&mat[i][j]);
        }
    }

    printf("\nNegativos:\n");

    for (int i=0; i<m; i++){
        for (int j=0; j<n; j++){
            if (mat[i][j] < 0){
                printf("%d\n",mat[i][j]);
            }
        }
    }

    return 0;
}

File: python/udemy/7-matrizes/negativos_matriz.py
# Ler dois números M e N (máximo = 10), e depois ler uma matriz MxN de números inteiros, conforme exemplo. Em seguida, mostrar na tela somente os números negativos da matriz.

m = int(input('Linhas da matriz (máximo 10): '))
n = int(input('Colunas da matriz (máximo 10): '))

mat = [[] for _ in range(m)]

for i in range(m):
    for j in range(n):
        valor = float(input(f'Elemento [{i}][{j}]: '))
        mat[i].append(valor)

print('Números negativos da matriz: ')
for i in range(m):
    for j in range(n):
        if mat[i][j] < 0:
            print(mat[i][j])

